cloudstate/cloud_state.py

```diff
@@ -22,12 +22,12 @@ def register_event_sourced_entity(self, entity: EventSourcedEntity):
         self.event_sourced_entities.append(entity)
         return self
 
-    def serve(self):
+    def start(self):
         server = grpc.server(futures.ThreadPoolExecutor(max_workers=10))
         add_EntityDiscoveryServicer_to_server(CloudStateEntityDiscoveryServicer(self.event_sourced_entities), server)
         add_EventSourcedServicer_to_server(CloudStateEventSourcedServicer(self.event_sourced_entities),server)
-        port = os.environ.get('HOST', '127.0.0.1') + ':' + os.environ.get('PORT', '50051')
+        port = os.environ.get('HOST', '127.0.0.1') + ':' + os.environ.get('PORT', '8080')
         server.add_insecure_port(port)
-        pprint('Starting CloudStateEntityDiscoveryServicer on ' + port)
+        pprint('Starting CloudState on ' + port)
```

Comment:
wording is Cloudstate, should we stick with that? I know we use **CloudState**EntityDiscoveryServicer, also in other repos, but we start "Cloudstate" here. wdyt @sleipnir

Reply:
I don't think we upload a service individually but the Cloudstate protocol itself. In other languages we see more logs of various parts of what is happening in the start phase but in the end what we are offering is the cloudstate itself

Reply:
oh I just wrote that because of upper lower case wording of Cloudstate, vs CloudState

Reply:
I understood that it was another question

Reply:
I think the good thing is how you like it best! I always write Cloudstate

Reply:
Like a God ;)

Reply:
I just strive for consistency... 😅

Reply:
Is that I understand it as a "Noun / Proper Name" and that's why I always use the first capital letter, but tell me how you think it should be and I make the adjustment

Reply:
Lightbend went from initially CloudState to Cloudstate to name it... that's the only reasoning…

Reply:
so I think we should write
`pprint('Starting Cloudstate on ' + port)`

Reply:
Just now I saw the capital letter 'S'

Reply:
mad max

Reply:
Fixed

Reply:
Awesome! Thanks @sleipnir. I appreciate any effort for perfection; always.